packages/faustwp-core/tests/utils/flatListToHierarchical.test.ts

```diff
@@ -0,0 +1,46 @@
+import flatListToHierarchical from '../../src/utils/flatListToHierarchical';
```

Comment:
We should import the utility function from packages/faustwp-core/src/utils/index.ts since it's available.

Reply:
This is the current pattern in place for the tests. They are using relative paths.